index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width", initial-scale=1.0>
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" href="http://fonts.googleapis.com/css?family=Roboto" >
+</head>
+<body>
+    <header>
+        <div>Ashmal Zahra</div>
```

Comment:
- [ ] I think your logo should be wrapped in an `a` tag with `href` attribute pointing to index.html. This is best for HTML semanatics and also great user experience since clicking on the logo will take you to the homepage.

- [ ] Please use an indentation of two spaces in all your documents for readability. This is a convention and also best practice.